test/cli.js

```diff
@@ -93,3 +94,57 @@ test('throwing a anonymous function will report the function to the console', fu
 		t.end();
 	});
 });
+
+test('Node.js-style --require CLI argument #1', function (t) {
```

Comment:
Instead of `#1`, `#2`, etc, it would be better if they just said specifically what they were testing differently than the others.